spec/requests/projects_api_spec.rb

```diff
@@ -55,9 +55,7 @@
           let(:resource) { project_stub }
           it 'should set creator to current_user and make them a project_admin, and queue an ActiveJob' do
             expect {
-              expect {
-                is_expected.to eq(201)
-              }.to have_enqueued_job(ProjectStorageProviderInitializationJob)
```

Comment:
Did you remove the `have_enqueued_job` test because the job is being enqueued in an `after_commit` callback? If so, it may be worth looking into the `test_after_commit` gem (until we upgrade to Rails 5), which is mentioned at the end of this article: 
http://www.justinweiss.com/articles/a-couple-callback-gotchas-and-a-rails-5-fix/